nickytonline/connect-four
# chore: add Gitpod support

## What type of PR is this? (check all applicable)


## Description

Adds support for working on the project in Gitpod.

TODO: Write docs related to this

## Related Tickets & Documents

- Related Issue #
- Closes #6

## QA Instructions, Screenshots, Recordings

1. To load GitPod for testing this PR, navigate to https://gitpod.io/#https://github.com/nickytonline/connect-four/tree/nickytonline/add-gitpod in your browser.

2. It'll take a minute to load up.

![image](https://user-images.githubusercontent.com/833231/193589795-cace26b2-a240-40a0-88ec-eab1a3542ba7.png)

4. Gitpod will prompt to open in VS Code. Click the X in the top right of the
 prompt.

![image](https://user-images.githubusercontent.com/833231/193589826-29769a2b-1931-4130-8439-b8cd353ed682.png)

5. Gitpod will prompt you to install the Svelte for VS Code extension. Click the Install button.

![image](https://user-images.githubusercontent.com/833231/193589867-74593b62-44ee-4707-bfcc-79e8a45d5324.png)

7. It will install all npm packages for the project. Once all the dependencies are installed, it will load the project by running `ntl dev` and the app will appear in development mode.

![image](https://user-images.githubusercontent.com/833231/193589737-fcc5fad8-654a-49ec-90ce-f9d4b6900eeb.png)

8. From there, you can start developing like you would if working on your local machine's IDE.

### UI accessibility concerns?

N/A

## Added/updated tests?

- [x] No, and this is why: Setting up a different environment for development

## [optional] What gif best describes this PR or how it makes you feel?

![Chartacter from Southpark saying, "Nice!"](https://media.giphy.com/media/pCO5tKdP22RC8/giphy.gif)
